packages/loader/container-loader/src/container.ts

```diff
@@ -662,20 +662,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this.connectionStateHandler.containerSaved();
         });
 
-        this._deltaManager = this.createDeltaManager();
```

Comment:
Why is this line deleted?

Reply:
Uhhhhhhhhhhhhhhh

Reply:
GitHub.com bug/feature I think.  Try refreshing the page.  I merged main and this file is no longer changed.

Reply:
I think I looked at just the changes between your last changes and your most recent changes.